cluster/ci/config/prow/plugins.yaml

```diff
@@ -34,6 +34,16 @@ config_updater:
       name: labels
     ci-operator/jobs/**/*.yaml:
       name: job-config
+    ci-operator/test-deploy/aws/*.yaml:
```

Comment:
Is there a reason to break these down by subdir instead of using `ci-operator/test-deploy/**/*.yaml` like you do for `job-config` above?  Using `**` seems like it might be less work to maintain as cluster profiles are added and removed, but maybe additions/removals don't happen often.